frontend/app_flowy/lib/workspace/presentation/stack_page/trash/trash_page.dart

```diff
@@ -124,7 +124,7 @@ class _TrashStackPageState extends State<TrashStackPage> {
             child: FlowyButton(
               text: const FlowyText.medium('Restore all', fontSize: 12),
               icon: svg('editor/restore'),
-              hoverColor: theme.hover,
+              hoverColor: theme.main2,
```

Comment:
theme.hover was the way to go. :) If the color needs to be dark, than it should be changed in the theme data, not here.